pkg/sriov/sriov.go

```diff
@@ -263,6 +263,26 @@ func (s *sriovManager) ApplyVFConfig(conf *types.NetConf) error {
 	return nil
 }
 
+// restoreVFName restores VF name from conf
+func (s *sriovManager) restoreVFName(conf *types.NetConf) error {
+	linkName, err := utils.GetVFLinkNames(conf.DeviceID)
+	if err != nil {
+		return fmt.Errorf("restoreVFName error: failed to get VF name, %v", err)
+	}
+
```

Comment:
can we only do the flow if linkName != conf.HostIFNames ?

Reply:
BTW, I forgot about it :/